app/jobs/scheduled/auto_queue_handler.rb

```diff
@@ -7,7 +7,7 @@ class AutoQueueHandler < ::Jobs::Scheduled
     every 1.day
 
     def execute(args)
-      return unless SiteSetting.auto_handle_queued_age.to_i > 0
+      return if SiteSetting.auto_handle_queued_age.to_i <= 0
```

Comment:
This already has `min 0` and 0 means disabled, so this can be `return if SiteSetting.auto_handle_queued_age.to_i.zero?`